platforms/documentation/docs/src/docs/css/base.css

```diff
@@ -41,7 +41,7 @@ a {
 }
 
 a:hover, a:focus {
-    text-decoration: underline;
+    text-decoration: none;
```

Comment:
🤔  This changes the style for all links, including inline links. I don't think it's the right choice. We only want the headers to not be underscored. Regular links in text should be underscored on hover as is usual on the internet

Reply:
Fixed.